### PR TITLE
Adjust query and filter in def _get_suggested_structures_for_channel

### PR DESCRIPTION
**Pull request by sjib**
filter on wastewater_structure.identifier instead of wastewater_networkelement (node).identifier

### CODE REVIEW

**Comment by sjib** [comment]
Should solve https://github.com/QGEP/qgepqwat2ili/issues/95

**Comment by sjib** [comment]
This solution will not cover the case that when you have a Kombischacht, that it matches right because then it is not clear which channel to take

File: qgepqwat2ili/gui/editors/examination.py [hunk 1]
```diff
@@ -173,15 +173,20 @@ def _get_suggested_structures_for_channel(self, from_id, to_id):
...
             .join(QGEP.reach)
             .join(rp_from, rp_from.obj_id == QGEP.reach.fk_reach_point_from)
             .join(wastewater_ne_from, wastewater_ne_from.obj_id == rp_from.fk_wastewater_networkelement)
+            .join(wastewater_st_from, wastewater_st_from.obj_id == wastewater_ne_from.fk_wastewater_structure)
             .join(rp_to, rp_to.obj_id == QGEP.reach.fk_reach_point_to)
             .join(wastewater_ne_to, wastewater_ne_to.obj_id == rp_to.fk_wastewater_networkelement)
-            .filter(wastewater_ne_from.identifier == from_id, wastewater_ne_to.identifier == to_id)
+            .join(wastewater_st_to, wastewater_st_to.obj_id == wastewater_ne_to.fk_wastewater_structure)
+            .filter(wastewater_st_from.identifier == from_id, wastewater_st_to.identifier == to_id)
+            #.filter(wastewater_ne_from.identifier == from_id, wastewater_ne_to.identifier == to_id)
```
Comment (olivierdalang) [inline qgepqwat2ili/gui/editors/examination.py:189]: remove leftover comment

**Review by olivierdalang — APPROVED** [review]
Looks good (besides the leftover comment).

This is not covered by automated tests, and I did not manually test, but if you did and it works as expected ok to merge.

**Pull request merged**